horace_core/sqw/page_operations/PageOpBase.m

```diff
@@ -193,11 +201,6 @@
             end
         end
         function obj = set.npix(obj,val)
-            if ~isnumeric(val)
```

Comment:
Why has this check been removed?

Reply:
I've wrote the description near the first comment.